api/server.go

```diff
@@ -24,11 +24,12 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/negroni"
+
 	"github.com/loadimpact/k6/api/common"
-	"github.com/loadimpact/k6/api/v1"
+	v1 "github.com/loadimpact/k6/api/v1"
```

Comment:
Why have the `v1` alias here, considering that it's identical to the module name?

Reply:
I get the same thing everytime .. I have the feeling it might have something to do with the fact that this is the same way that go modules version should be specified ? So the problem is that the package name is not `api` ?

Reply:
@na-- @MStoykov yes, without the alias, when switching to go modules, `"github.com/loadimpact/k6/api/v1"` can be mis-interpreted as package `api`, using v1, instead of package `v1`.

Anyway, it's the behavior of `goimports`, the line was not modified by me :smile:

Reply:
Found the reason https://go-review.googlesource.com/c/tools/+/152000